sklearn/preprocessing/_encoders.py

```diff
@@ -218,6 +218,18 @@ class OneHotEncoder(_BaseEncoder):
             The ``n_values_`` attribute was deprecated in version
             0.20 and will be removed in 0.22.
 
+    handle_missing : all-missing, all-zero or category
+        What should be done to missing values. Should be one of:
+
+        all-missing: Replace with a row of NaNs as above
+
+        all-zero: Replace with a row of zeros
+
+        category: Represent with a separate one-hot column
+
+    missing_values: NaN or None
```

Comment:
I still think we would be best off supporting only NaN initially. Simplifies the code and the reviewing... and handling NaN and None properly is tricky.

Reply:
> I still think we would be best off supporting only NaN initially. Simplifies the code and the reviewing... and handling NaN and None properly is tricky.

\+1